client/state/jitm/actions.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Internal Dependencies
+ */
+import { JITM_DISMISS, JITM_SET } from 'state/action-types.js';
+
+export const dismissJetpackJITM = ( dispatch, siteId, id, featureClass ) => {
+	dispatch(
+		{
+			type: JITM_DISMISS,
+			siteId,
+			id,
+			featureClass,
+		}
+	);
+};
```

Comment:
This is something we would probably be caring more about to make sure we don't dispatching from action creators. Instead we'd much rather have the action creators remain simple functions which return action objects and let someone else handle the dispatch.